flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamContextEnvironment.java

```diff
@@ -38,14 +38,10 @@
 	private final ContextEnvironment ctx;
 
 	protected StreamContextEnvironment(ContextEnvironment ctx) {
+		super(GlobalConfiguration.loadConfiguration().getInteger(
```

Comment:
I had a quick offline chat with @StephanEwen and he agreed that we should address this in a follow-up PR. I created a new Jira issue: https://issues.apache.org/jira/browse/FLINK-6058.